packages/cli/src/cli/cogify/action.make.cog.ts

```diff
@@ -192,7 +192,7 @@ export class CommandMakeCog extends CommandLineAction {
         id,
         projection: Epsg.Nztm2000,
         resampling,
-        alignedLevel: this.adjustAlignedLevel(tileMatrix, alignedLevel),
+        alignedLevel: alignedLevel,
```

Comment:
NIT: you dont need to specify  `: alignedLevel`

Reply:
Ahh good point. Updated.